.gitignore

```diff
@@ -218,3 +218,4 @@ empty_test_environment_variables.py
 /cloud_governance/policy/send_mail.py
 cloudsensei/.env.txt
 .vscode
+cloud_governance/main/env.yaml
```

Comment:
Where did u put this file

Reply:
inside the main folder

Reply:
Sorry I dont see it

Reply:
I added to .gitignore file that's why it is hidden.

Reply:
@athiruma ,
`cloud_governance/main/env.yaml`: remove from .gitignore and it should be filled with default/empty values
`cloud_governance/main/test_env.yaml`: add to .gitignore and it should be filled with real values

Reply:
No need to maintain 2 files, we can use test_env.yaml. In the readme, we will mention creating a copy of
`cp test_env.yaml env_yaml` and change appropriate values.